pkgs/by-name/wo/workout-tracker/package.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  nix-update-script,
+  ...
```

Comment:
```suggestion
```

Reply:
@wegank pardon my insolence, but perhaps these kind of trivial automated suggestions should be part of CI, and if not, they should be made before someone has spent multiple hours on addressing thoughtful feedback made by actual humans.

Also, this comment gives no indication why this change must be made, and frankly, at this point I do not want to.

@leona-ya my simcerest apologies if not making this change is a blocker and I've wasted your time in that case, please let me know and I'll just close the PR.

Reply:
I'm surprised to read this at 4 am in my time zone, I just got off my bed to say that I'm not a bot, and my suggestion isn't automated. It is part of a review following [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md), if you look for "Functions should list their expected arguments as precisely as possible". I'm sorry if that wasn't clear.

Anyway, I don't want you to close the PR, so I'll merge it, and you're free to make the change in a PR or not afterwards.

Reply:
Hi @wegank ,

First of all, my comment was unnecessarily rude to you, it was wrong and I am sorry for it. Lesson learned, wait and think before putting finger to keyboard.

It has been a while since I've read contributing guide and this is perhaps good reminder to do it again.

Coming back, and I really mean this with respect, that I've had such tiny, most definitely automated review comments in past, although never from you. I understand it is more work, but above reason with link to contributing guide would have been much much better, as it tells me why something is being asked of me, and help understand it wasn't just somebody being pedantic.

Ultimately, as with nixfmt, I sincerely think things, in grand scheme of things, should be automated. I was very relieved to not have to live through formatting comments, and a fix like current one can be trivially performed via a linter. If you have a preferred one, can you please link me to it so I can be more careful from next time onwards?

P.S.: Thanks for merging, I'll be sure to fix this in next commit.

Reply:
> I understand it is more work, but above reason with link to contributing guide would have been much much better, as it tells me why something is being asked of me, and help understand it wasn't just somebody being pedantic.

I'll keep that in mind, thanks.

> If you have a preferred one, can you please link me to it so I can be more careful from next time onwards?

I don't use any Nix linters, except perhaps nil which does give some lints for my own PRs, so I can't help you there...